src/ws/cockpitwebservice.c

```diff
@@ -929,12 +931,12 @@ process_logout (CockpitWebService *self,
   /* Destroys our web service, disconnects everything */
   if (disconnect)
     {
-      g_info ("Logging out session of user %s from %s", cockpit_creds_get_user (self->creds), cockpit_creds_get_rhost (self->creds));
+      g_info ("Logging out session %s", self->id);
```

Comment:
Nitpick: It was nice to see the user name in the log, as after the session ends, it can't be reconstructed from loginctl any more.

Reply:
It's still in the journal from when the session has started, no?

Reply:
You're totally right, of course!